src/SDKs/SqlManagement/Sql.Tests/SyncGroupScenarioTests.cs

```diff
@@ -0,0 +1,117 @@
+                {
+                    Location = server.Location,
+                });
+                Assert.NotNull(syncDatabase);
+                Assert.NotNull(syncDatabase.Id);
+
+                // Create database
+                string testDatabaseName = SqlManagementTestUtilities.GenerateName(testPrefix + "test");
+                Database testDatabase = sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, server.Name, testDatabaseName, new Database()
+                {
+                    Location = server.Location,
+                });
+                Assert.NotNull(testDatabase);
+
+                // Create sync group
+                string syncGroupName = SqlManagementTestUtilities.GenerateName(testPrefix + "syncgroup");
+                int interval1 = -1; // Manual
+                string conflictPolicy = SyncConflictResolutionPolicy.MemberWin;
+                SyncGroup createSyncGroup = sqlClient.SyncGroups.CreateOrUpdate(resourceGroup.Name, server.Name, testDatabaseName, syncGroupName, new SyncGroup
+                {
+                    Interval = interval1,
+                    ConflictResolutionPolicy = conflictPolicy,
+                    SyncDatabaseId = syncDatabase.Id,
+                    HubDatabaseUserName = SqlManagementTestUtilities.DefaultLogin,
+                    HubDatabasePassword = SqlManagementTestUtilities.DefaultPassword
+                });
+                Assert.NotNull(createSyncGroup);
+                Assert.Equal(interval1, createSyncGroup.Interval);
+                Assert.Equal(conflictPolicy, createSyncGroup.ConflictResolutionPolicy);
+                Assert.Equal(syncDatabase.Id, createSyncGroup.SyncDatabaseId);
+
+                // Get sync group
+                SyncGroup getSyncGroup = sqlClient.SyncGroups.Get(resourceGroup.Name, server.Name, testDatabaseName, syncGroupName);
+                Assert.NotNull(getSyncGroup);
+                Assert.Equal(interval1, getSyncGroup.Interval);
+                Assert.Equal(conflictPolicy, getSyncGroup.ConflictResolutionPolicy);
+                Assert.Equal(syncDatabase.Id, getSyncGroup.SyncDatabaseId);
+
+                // List sync group
+                IPage<SyncGroup> listSyncGroups = sqlClient.SyncGroups.ListByDatabase(resourceGroup.Name, server.Name, testDatabaseName);
+                Assert.NotNull(listSyncGroups);
+                Assert.Equal(1, listSyncGroups.Count());
+                Assert.Equal(syncGroupName, listSyncGroups.FirstOrDefault()?.Name);
+
+                // Update sync group
+                int interval2 = 600;
+                SyncGroup updateSyncGroup = sqlClient.SyncGroups.Update(resourceGroup.Name, server.Name, testDatabaseName, syncGroupName, new SyncGroup
+                {
+                    Interval = interval2
+                });
+                Assert.NotNull(updateSyncGroup);
+                Assert.Equal(interval2, updateSyncGroup.Interval);
```

Comment:
oops, these empty updates ideally should not fail. Not your fault though, it has to be fixed server-side. Thanks for testing!